jetty-home/pom.xml

```diff
@@ -402,33 +402,6 @@
               <outputDirectory>${source-assembly-directory}/lib/glassfish-jstl</outputDirectory>
             </configuration>
           </execution>
-          <execution>
```

Comment:
We still need to copy the jaspi dependency jars into ${assembly-directory/lib/jaspi.

Reply:
this seems to only include to be the orbit jars that are no longer needed?

Reply:
I'm out of my depth here, but there are several things that need to be done.  I've tried to illustrate using the following mock jaspi-basic.xml file

```
<?xml version="1.0"?>
<!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "http://www.eclipse.org/jetty/configure_10_0.dtd">

<Configure id="context" class="org.eclipse.jetty.webapp.WebAppContext">

  <!-- Step 1: init AuthConfigFactory for jakarta. 
       TBD: Could alternatively be set through system property "authconfigprovider.factory" defined by jakarta.security.auth.message.config.AuthConfigFactory#DEFAULT_FACTORY_SECURITY_PROPERTY and referenced from AuthConfigFactory#getFactory
  -->
  <New id="jaspiAuthConfigFactory" class="org.eclipse.jetty.jaspi.BasicAuthConfigFactory" />
  <Call class="jakarta.security.auth.message.config.AuthConfigFactory" name="setFactory">
    <Arg><Ref refid="jaspiAuthConfigFactory" /></Arg>
  </Call>
  
  <!-- Step 2: configure AuthConfigProvider with appropriate ServerAuthModules -->
  <Ref refid="jaspiAuthConfigFactory">
	<Call name="registerConfigProvider">
		<Arg type="String">org.eclipse.jetty.security.jaspi.BasicAuthModule</Arg>
	    <Arg>
            <Map>
              <Entry>
                <Item>AppContextID</Item>
                <Item>server /other</Item>
              </Entry>
			  <Entry>
				<Item>org.eclipse.jetty.security.jaspi.modules.RealmName</Item>
				<Item>TestRealm</Item>
			  </Entry>
            </Map>
		</Arg>
		<Arg type="String">HttpServlet</Arg>
		<Arg type="String">server /ctx</Arg>
		<Arg type="String">a test provider</Arg>
	</Call>
	<Call name="registerConfigProvider">
		<Arg type="String">org.eclipse.jetty.security.jaspi.HttpHeaderAuthModule</Arg>
		<Arg>
            <Map>
              <Entry>
                <Item>AppContextID</Item>
                <Item>server /other</Item>
              </Entry>
            </Map>
		</Arg>
		<Arg type="String">HttpServlet</Arg>
		<Arg type="String">server /other</Arg>
		<Arg type="String">another test provider</Arg>
	</Call>
  </Ref>
  
  <!-- Step 3: connect JASPI AuthConfigFactory to WebAppContext -->
  <Set name="securityHandler">
    <New class="org.eclipse.jetty.security.ConstraintSecurityHandler">
      <Set name="loginService">
        <New class="org.eclipse.jetty.plus.jaas.JAASLoginService">
          <Set name="name">JAASRealm</Set>
          <Set name="loginModuleName">jaas</Set>
        </New>
      </Set>
  
      <Set name="authenticatorFactory">
        <New class="org.eclipse.jetty.security.jaspi.JaspiAuthenticatorFactory" />
      </Set>
    </New>
  </Set>
  
</Configure>

```

Step 1:  This sets up the BasicAuthConfigFactory to be de default AuthConfigFactory.  Can alternatively be done through system property

Reply:
Sure but don't we need to copy in the _new_ jaspi dependency jars into `${assembly-directory/lib/jaspi`?

Reply:
ok, adding jakarta.authentication-api here (only spec needed, no other deps)